torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -958,7 +958,9 @@ RegisterOperators reg2({
     DEFINE_INT_OP(aten::__lshift__, a << b),
     DEFINE_INT_OP(aten::__rshift__, a >> b),
 
-    DEFINE_UNARY_OP(aten::round, std::round(a), float, float),
+    // use this trick to enforce rounding to even.
+    // NOLINTNEXTLINE(cppcoreguidelines-avoid-magic-numbers)
+    DEFINE_UNARY_OP(aten::round, std::round(a * 0.5) * 2.0, float, float),
```

Comment:
How about the case say, where a = 0.8?

in python `round(0.8) = 1`
but `std::round(0.8*0.5) * 2 = 0`

Reply:
should be good now